tests/lib/cli-engine/formatters/stylish.js

```diff
@@ -127,15 +89,17 @@ describe("formatter:stylish", () => {
 			});
 
 			it("should return a string in the correct format", () => {
-				const result = formatter(code);
+				const result = util.stripVTControlCharacters(formatter(code));
 
 				assert.strictEqual(
 					result,
 					"\nfoo.js\n  5:10  error  Unexpected foo  foo\n\n\u2716 1 problem (1 error, 0 warnings)\n  1 error and 0 warnings potentially fixable with the `--fix` option.\n",
 				);
-				assert.strictEqual(chalkStub.reset.callCount, 1);
-				assert.strictEqual(chalkStub.yellow.bold.callCount, 0);
-				assert.strictEqual(chalkStub.red.bold.callCount, 2);
+				/*
+				 * assert.strictEqual(chalkStub.reset.callCount, 1);
+				 * assert.strictEqual(chalkStub.yellow.bold.callCount, 0);
+				 * assert.strictEqual(chalkStub.red.bold.callCount, 2);
+				 */
```

Comment:
To keep the assertions:

```suggestion
			assert(util.styleText.calledWith("reset"));
			assert(util.styleText.neverCalledWith("yellow"));
			assert(util.styleText.calledWith("red"));
```

No trivial way to assert that `["red", "bold"]` was used twice. If the invocation order is assumed, a particular call can be checked by its index:

```js
assert(util.styleText.getCall(4).calledWith(["red", "bold"]));
assert(util.styleText.getCall(5).calledWith(["red", "bold"]));
```

Similarly for the other test cases.